src/group_theory/quotient_group.lean

```diff
@@ -412,4 +413,44 @@ monoid_hom.to_mul_equiv
 
 end third_iso_thm
 
+
+section trivial
+
+/-These first few lemmas should maybe go somewhere else, where?-/
+
+lemma quot_by_top_is_subsingleton : subsingleton (quotient_group.quotient (⊤ : subgroup G)) :=
+trunc.subsingleton
+
+lemma left_rel_triv (H : subgroup G) (h : (quotient_group.left_rel H).r = λ _ _, true  ) :
```

Comment:
I'd be inclined to state this as:
```suggestion
lemma left_rel_triv (H : subgroup G) (h : quotient_group.left_rel H = ⊤) :
```

Reply:
Yeah I think thats nicer. I've done this.